docs/source/javascripts/site.js

```diff
@@ -71,6 +82,43 @@ window.modules.search = (function () {
             return list;
         },
 
+        constructPluginResultList = function (results) {
+            var plugin, list = '';
+            for (var i = 0; i < results.length; i++) {
+                plugin = results[i];
+                list += pluginResultListItemTPL(plugin.name, plugin.git_hub_key);
+            }
+
+            return list;
+        },
+
+        pluginResult = function (query) {
+            var results = [],
+                plugins = [
+                    {
```

Comment:
RFC: A better place to put this list?

Reply:
It would be _really_ neat if we could pull JSON results from https://plugins.workarea.com, but I just checked and it doesn't seem there's a JSON response for `/plugins`...

Reply:
How about using a constant-like var in this file? So at least it won't make this function super long to read.